sdk/servicebus/service-bus/src/connectionContext.ts

```diff
@@ -617,37 +617,20 @@ export namespace ConnectionContext {
     try {
       logger.verbose(`${logPrefix} Permanently closing the amqp connection on the client.`);
 
-      // Close all the senders.
       const senderNames = Object.keys(context.senders);
-      logger.verbose(`${logPrefix} Permanently closing ${senderNames.length} senders.`);
-      for (const senderName of senderNames) {
-        await context.senders[senderName].close();
-      }
-
-      // Close all MessageReceiver instances
       const messageReceiverNames = Object.keys(context.messageReceivers);
-      logger.verbose(`${logPrefix} Permanently closing ${messageReceiverNames.length} receivers.`);
-      for (const receiverName of messageReceiverNames) {
-        await context.messageReceivers[receiverName].close();
-      }
-
-      // Close all MessageSession instances
       const messageSessionNames = Object.keys(context.messageSessions);
-      logger.verbose(
-        `${logPrefix} Permanently closing ${messageSessionNames.length} session receivers.`
-      );
-      for (const messageSessionName of messageSessionNames) {
-        await context.messageSessions[messageSessionName].close();
-      }
 
-      // Close all the ManagementClients.
-      const managementClientsEntityPaths = Object.keys(context.managementClients);
       logger.verbose(
-        `${logPrefix} Permanently closing ${managementClientsEntityPaths.length} session receivers.`
+        `${logPrefix} Permanently closing all the senders, MessageReceivers, MessageSessions, and ManagementClients.`
```

Comment:
Can we include the count in paranthesis perhaps? That way we retain that info?

`Permanently closing all the senders(4), MessageReceivers(3), MessageSessions(0), and ManagementClients(1)`

Reply:
yes that could be useful.